### PR TITLE
Issue 45 template fix

### LINKED ISSUE #45: Template fix
Template for the PR uses "Title: "Closes issue #<number> " which **doesn't close the issue when the PR gets merged**, change wording to "Closes #<number>".

ALso, remove some part of the PR template:
- [x] remove the title, about, labels, and assignees section - I don't think these do anything?
- [x] Change "What does this implement/fix?" to "summary of changes"
- [x] Reduce number of checklist items 
    - remove: self-review, change to documentation, generate no new warnings, dependency change
    - merge the two testing items into one item
    - add type annotation

Also is there a template for issues? A simple template would probably do

### PR DESCRIPTION
**Pull request by ctmes**
---
Title: "Closes #45"
---

# **Summary of Changes**
Fix verbose and unnecessary parts of old pr template

### **Optional Further Details**
Refactor checklist
Reduce table

**Currently issue templates aren't working, I assume because GitHub hasn't fully finished polishing them**

### **Screenshots**
If applicable, add screenshots to help visualize the changes.

# **Checklist:**
- [x] My code follows the code style of this project.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] New and existing unit tests prove my fix is effective or that my feature works.
- [x] I have annotated added types appropriately.

### **Additional context**
Add any other context or relevant information about the pull request here.

### CODE REVIEW

**Comment by ctmes** [comment]
Created config.yml which will _hopefully_ fix issue templates.

File: .github/ISSUE_TEMPLATE/1-DEV_ISSUE.md [hunk 1]
```diff
@@ -1,31 +1,21 @@
...
 ---
 
-## **Summary**
-Briefly describe the issue or task.
-
-## **Steps to Reproduce**
-If applicable, list key steps to replicate the issue.
+# Issue
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is.
```
Comment (SodaVolcano) [inline .github/ISSUE_TEMPLATE/1-DEV_ISSUE.md:8]: Can simplify to just `# Summary of the issue/bug/feature request`

File: .github/ISSUE_TEMPLATE/1-DEV_ISSUE.md [hunk 2]
```diff
@@ -1,31 +1,21 @@
...
 
-## **Technical Details**
-Key components/files affected, relevant logs, or errors.
+## Suggested Solution
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
```
Comment (SodaVolcano) [inline .github/ISSUE_TEMPLATE/1-DEV_ISSUE.md:12]: Can just write `# Suggested Solution/Tasks`

File: .github/ISSUE_TEMPLATE/1-DEV_ISSUE.md [hunk 3]
```diff
@@ -1,31 +1,21 @@
...
 
-## **Acceptance Criteria**
-- [ ] Criteria 1
-- [ ] Criteria 2
-- [ ] Criteria 3
+## Alternatives Considered
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
```
Comment (SodaVolcano) [inline .github/ISSUE_TEMPLATE/1-DEV_ISSUE.md:16]: remove, most things on here are meant for newcomers to the repository which doesn't apply to us

**Review by SodaVolcano — CHANGES_REQUESTED** [review]
looks good to me just need to remove a few more lines :ok_hand: :shrimp: :shrimp:

**Comment by SodaVolcano** [comment]
btw @ctmes the issue template is working now (it wasn't working at the time I posted #45 ), see #47

**Comment by ctmes** [comment]
> btw @ctmes the issue template is working now (it wasn't working at the time I posted #45 ), see #47

Did you want me to work on #47 too?

File: .github/ISSUE_TEMPLATE/1-DEV_ISSUE.md (hunk 3 above)
Comment (ctmes) [reply .github/ISSUE_TEMPLATE/1-DEV_ISSUE.md]: done!

**Review by SodaVolcano — APPROVED** [review]

**Comment by SodaVolcano** [comment]
> Did you want me to work on #47 too?

If you want to look into that that'll be great, but that issue (and this issue) are more for convenience so we don't have to do them manually. If these Github-related issues are taking up too much of your time then it's fine we leave them unresolved. They exist to save some time but we don't want to sink too much time into setting them up in the first place :sweat:

**Review by JessicaShindunata — APPROVED** [review]

**Pull request merged**